aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/JobsServiceImpl.java

```diff
@@ -226,9 +203,12 @@ public String startRunReScanApplication(String applicationGuid, String nodeName,
         if (StringUtils.isNotEmpty(snapshotName)) {
             requestBuilder.snapshotName(snapshotName);
         }
-
+        if (moduleGenerationType != null) {
+            requestBuilder.moduleGenerationType(moduleGenerationType.toString());
```

Comment:
another PR needed

Reply:
And what?

Reply:
create another PR with right ticket for that